basics/getting_started.md

```diff
@@ -135,7 +135,7 @@ Bazel is organized as follows:
      utilities.
    *  Built-in rules are in `com.google.devtools.build.lib.rules` and in
      `com.google.devtools.build.lib.bazel.rules`. You might want to read about
-     the [Challenges of Writing Rules](docs/rule-challenges.html) first.
+     the [Challenges of Writing Rules](https://docs.bazel.build/versions/master/rule-challenges.html) first.
```

Comment:
@communikit I wonder if we should, instead, move the rule-challenges page to this site. 
- it is canonical advice, not tied to a specific bazel version.
- it is not referenced from anywhere else in the bazel docs.

Reply:
I think it is fine to stay where it is for now, we can overhaul moving pages around in a separate effort.